NuKeeper.Inspection/Logging/IInternalLogger.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace NuKeeper.Inspection.Logging
+{
+    public interface IInternalLogger
+    {
+        void Error(string message, Exception ex = null);
```

Comment:
Personal bias against optional parameters, but: this makes it way easier to forget to include the actual exception when logging. I'd vote for explicit `null` in those cases.

Reply:
Yep, the default is no longer needed on the internal logger.